src/args.rs

```diff
@@ -0,0 +1,29 @@
+use clap:: {
+    Parser,
+};
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
+pub struct PokerArgs{
+    // Option command to run the program
+    #[arg(short = 'r', long = "run")]
```

Comment:
This one is confusing. In practice, it seems like if the user sets `-r`, then it runs everything, otherwise it just prints the deck.